src/psyclone/psyir/backend/opencl.py

```diff
@@ -162,9 +162,18 @@ def kernelschedule_node(self, node):
         data_args = symtab.data_arguments
 
         # Start OpenCL kernel definition
-        code = self._nindent + "__kernel void " + node.name + "(\n"
+        code = self._nindent
+        if node._opencl_options['local_size'] != 1:
+            loc_size_value = node._opencl_options['local_size']
+            code += "__attribute__((vec_type_hint(double)))\n" \
+                    "".format(loc_size_value)
```

Comment:
You're using `.format()` here but there's no e.g. `{}` or `{0}` in the string itself.

Reply:
Ups, fixed.